source/promap.js

```diff
@@ -0,0 +1,9 @@
+import _curry3 from './internal/_curry3';
+import _dispatchable from './internal/_dispatchable';
+import _promap from './internal/_promap';
+import _xpromap from './internal/_xpromap';
+
+
+var promap = _curry3(_dispatchable(['fantasy-land/promap', 'promap'], _xpromap, _promap));
```

Comment:
Perhaps this could include some JSDoc and examples?

Reply:
Yes, it should.  I looked at the test cases, and didn't even notice that the JSDocs were missing!

Reply:
Oh, that's my bad; I misread the contributing guidelines saying to exclude `@since` as excluding JSDoc annotations - will get them in today!

Reply:
added JSDocs 👍